packages/wonder-blocks-tooltip/components/tooltip.js

```diff
@@ -175,20 +158,27 @@ export default class Tooltip extends React.Component<Props, State> {
     }
 
     _renderTooltipAnchor(ids?: IIdentifierFactory) {
-        const {forceAnchorFocusivity} = this.props;
+        const {children, forceAnchorFocusivity} = this.props;
+        const {active, anchorElement} = this.state;
+
+        const popperHost =
+            maybeGetPortalMountedModalHostElement(anchorElement) ||
+            document.querySelector("body");
```

Comment:
Probably better to just do `document.body` -- if you're worried about it being null then `document.querySelector` isn't going to be better (from a Flow perspective).

Reply:
yeah, good point